src/overlays/actors/ovl_Bg_Astr_Bombwall/z_bg_astr_bombwall.c

```diff
@@ -86,9 +86,7 @@ void BgAstrBombwall_InitCollider(ColliderTrisInit* init, Vec3f* pos, Vec3s* rot,
 
     for (i = 0; i < init->count; i++) {
         for (j = 0; j < 3; j++) {
-            //! FAKE:
-            // https://decomp.me/scratch/JrEnl
-            Matrix_MultVec3f(&(init->elements + i)->dim.vtx[j], &sp54[j]);
+            Matrix_MultVec3f(init->elements[i].dim.vtx + j, &sp54[j]);
```

Comment:
Looks better (though might still be considered a fake). Note this matches what was already in `func_809CE068` in `func_809CE068`